tutorials/pause_run_simulation.md

```diff
@@ -0,0 +1,57 @@
+\page pause_run_simulation Pause and Run simulation
+
+The TODO Gazebo transport API is exposed to allow starting and stopping the simulation.
```

Comment:
Please suggest the right name. I wasn't sure how to easily figure this out.

Reply:
You may just say "a Gazebo Transport API"

Reply:
```suggestion
A Gazebo transport API is exposed to allow starting and stopping the simulation.
```